.github/workflows/branch-pr-release.yaml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  release:
+    types:
+    - created
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+    - run: npm install
+    - run: xvfb-run -a npm test
+      if: runner.os == 'Linux'
+    - run: npm test
+      if: runner.os != 'Linux'
+    - name: Publish
+      if: success() && startsWith( github.ref, 'refs/tags/releases/') && matrix.os == 'ubuntu-latest'
+      run: vsce publish -p ${{ secrets.VSCE_PAT }}
```

Comment:
How should we handle auto-incrementing version numbers from this command?

Reply:
Need to check this but typically vsce will create a commit and do the auto-incrementing.  I would guess this mechanism will not do that work and just publish the latest tag.

Reply:
This will be based on the tag. So we still need to do the tagging